po/sv.po

```diff
@@ -549,39 +548,34 @@ msgstr ""
 "nätverkstid ska vara aktiverat."
 
 #: ../src/core/dbus-unit.c:428
-#, fuzzy
 msgid "Authentication is required to start '$(unit)'."
-msgstr "Autentisering krävs för ange systemtiden."
+msgstr "Autentisering krävs för att starta '$(unit)'."
 
 #: ../src/core/dbus-unit.c:429
-#, fuzzy
 msgid "Authentication is required to stop '$(unit)'."
-msgstr "Autentisering krävs för ange systemtiden."
+msgstr "Autentisering krävs för att stoppa '$(unit)'."
 
 #: ../src/core/dbus-unit.c:430
-#, fuzzy
 msgid "Authentication is required to reload '$(unit)'."
-msgstr "Autentisering krävs för att läsa om tillståndet för systemd."
+msgstr "Autentisering krävs för att läsa om tillståndet för '$(unit)'."
 
 #: ../src/core/dbus-unit.c:431 ../src/core/dbus-unit.c:432
-#, fuzzy
 msgid "Authentication is required to restart '$(unit)'."
-msgstr "Autentisering krävs för ange systemtiden."
+msgstr "Autentisering krävs för att starta om '$(unit)'."
 
 #: ../src/core/dbus-unit.c:535
-#, fuzzy
 msgid "Authentication is required to kill '$(unit)'."
-msgstr "Autentisering krävs för att logga till en lokal behållare"
+msgstr "Autentisering krävs för att döda '$(unit)'."
 
 #: ../src/core/dbus-unit.c:565
-#, fuzzy
 msgid "Authentication is required to reset the \"failed\" state of '$(unit)'."
-msgstr "Autentisering krävs för att ange lokalt värdnamn."
+msgstr ""
+"Autentisering krävs för att återställa det \"falerade\" tillståndet för "
+"'$(unit)'."
 
 #: ../src/core/dbus-unit.c:597
-#, fuzzy
 msgid "Authentication is required to set properties on '$(unit)'."
-msgstr "Autentisering krävs för ange systemtiden."
+msgstr "Autentisering krävs för att ange egenskaper på '$(unit)'."
```

Comment:
Does it make more sense to you with `ställa` instead of `ange`? It seems as if this it the normal way to translate this the rest of the translation.